### ISSUE TITLE
Searching for Youtube videos in your own room doesn't work (screenshot)

### ISSUE THREAD

**Issue by brace110** [Mewte/InstaSync#2]
![screen shot 2018-01-20 at 02 46 11](https://user-images.githubusercontent.com/1846025/35178655-3a86d366-fd8c-11e7-9b0f-eb618fa58fb0.png)

Searching for videos to add to your own playlist doesn't result in any results. 403 error, see screenshot.

**Comment by Mewte** [comment]
yeah its API token related, I need to create an API endpoint that handles the search function so that I don't have to put the token client side.

I'll add it to the list of things I need to do
